message.go

```diff
@@ -304,7 +307,9 @@ func (cli *Client) decryptMessages(info *types.MessageInfo, node *waBinary.Node)
 			return
 		}
 		retryCount := ag.OptionalInt("count")
-		cli.cancelDelayedRequestFromPhone(info.ID)
+		if retryCount > 0 {
+			cli.cancelDelayedRequestFromPhone(info.ID)
+		}
```

Comment:
`cancelDelayedRequestFromPhone` is a no-op if there's no pending request, so there's no need to check the retry count here